projects/ngx-feature-flag/src/lib/warning-page/feature-warning-page.component.html

```diff
@@ -1,50 +1,24 @@
-<div [ngSwitch]="level">
-  <div *ngSwitchCase="'internal'">
-    <div class="container-fluid content padding-top-15 internal-page internal-border">
-      <div class="col-xs-12 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 feature-content">
-        <div class="col-xs-12">
-          <i class="fa fa-lock fa-5x"></i>
-          <h2>Internal Features Opt-in</h2>
-          <p>These features are only available to Red Hat users and have no guarantee of performance or stability.
-            Use these at your own risk.</p>
-          <button class="btn btn-lg btn-feature-level" [routerLink]="profileSettingsLink">Opt-in to Internal Features</button>
-         </div>
-      </div>
-    </div>
-  </div>
-  <div *ngSwitchCase="'experimental'">
-    <div class="container-fluid content padding-top-15 experimental-page experimental-border">
-      <div class="col-xs-12 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 feature-content">
-        <div class="col-xs-12">
-          <i class="fa fa-flask fa-5x"></i>
-          <h2>Experimental Features Opt-in</h2>
-          <p>These features are currently in beta testing and have no guarantee of performance or stability.
-            Use these at your own risk.</p>
-            <button class="btn btn-lg btn-feature-level" [routerLink]="profileSettingsLink">Opt-in to Experimental Features</button>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div *ngSwitchCase="'beta'">
-    <div class="container-fluid content padding-top-15 beta-page beta-border">
-      <div class="col-xs-12 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 feature-content">
-        <div class="col-xs-12">
-          <i class="fa fa-5x">&Beta;</i>
-          <h2>Beta Features Opt-in</h2>
-          <p>These features are currently in beta testing and have no guarantee of performance or stability.
-            Use these at your own risk.</p>
-          <button class="btn btn-lg btn-feature-level" [routerLink]="profileSettingsLink">Opt-in to Beta Features</button>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div *ngSwitchCase="'notApplicable'">
-    <div class="container-fluid content padding-top-15 internal-page not-applicable-border">
-      <div class="col-xs-12 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 feature-content">
-        <div class="col-xs-12">
-          <i class="fa fa-flask fa-5x"></i>
-          <h2>Ooops</h2>
-          <p>This feature has been disabled by your admin page.</p>
+<div *ngIf="featureWarning" class="container-fluid container-cards-pf">
```

Comment:
@divyanshiGupta Is this backwards compatible with the way it was being used earlier with `FeatureFlagResolver`?

Reply:
Yes @rohitkrai03 this is backwards compatible, I have implemented this in a way that it doesn't breaks when used with `FeatureFlagResolver`. I have also tested it with a feature using `FeatureFlagResolver` at the route level.

Reply:
That's good. Nice work there.